docs/io/fits/index.rst

```diff
@@ -706,10 +755,8 @@ further simplify to just one call, instead of two as shown in the above
 examples::
 
     >>> from astropy.io.fits import getval
-    >>> flt = getval('in.fits', 'filter', 1)  # get 1st extension's keyword
-                                              # FILTER's value
-    >>> val = getval('in.fits', 10, 'sci', 2)  # get the 2nd sci extension's
-                                               # 11th keyword's value
+    >>> flt = getval(fits_image_2_filename, 'filter', 0)   # get 0th extension's keyword FILTER's value
+    >>> val = getval(fits_image_2_filename, 10, 'sci', 2)  # get the 2nd sci extension's 11th keyword's value
```

Comment:
Output value are not verified here, I understand why it's not done above for complete headers but here it is just a value so it would be easy to do.